src/Microsoft.AspNetCore.Proxy/ProxyMiddleware.cs

```diff
@@ -1,22 +1,28 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using Microsoft.AspNetCore.Builder;
```

Comment:
Sorry if this wasn't clear earlier but we put the all System namespaces before the Microsoft namespaces. After this, it should be good to merge.